docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+    * ``projector_ratio``: the ratio for the projector, e.g. ``16:9``.
+* ``cfp``: call for papers details. Contains:
+    * ``url``: the URL to a google form with the CFP.
+    * ``ends``: a human readable date of when the CFP ends,
+      e.g. ``31 January, 2020``.
+    * ``notification``: a human readable date of when accepted speakers
+      will be notified.
+* ``job_fair.location``: a description of the location of the job fair.
+
+Then, there are a number of true/false flags. Some of these don't change
+as the conference is being planned. These are:
+
+* ``flaghashike``: does the conference have a hike?
+* ``flaghasboat``: does the conference have a boat ride?
+* ``flagwelcomewagon``: does the conference have a welcome wagon?
+
+Others will change over time:
+
+* ``flaglanding``: is the conference website only a landing page?
+  Typically used for an early announcement of conference dates.
+* ``flagticketsonsale``: are tickets on sale at this time?
+* ``flagsoldout``: is the conference sold out?
+* ``flagcfp``: is the CFP currently open?
+* ``flagspeakersannounced``: have speakers been announced?
+  This flag requires the ``_data/<shortcode>-<year>-sessions.yaml``
+  file to exist, which lists the speakers/sessions.
+* ``flaghasschedule``: is the schedule ready?
+  This flag requires the``_data/<shortcode>-<year>-schedule.yaml``
+  file to exist, which contains the schedule.
+* ``flaglivestreaming``: is a live stream currently running?
+* ``flagpostconf``: has the conference ended?
+* ``flagvideos``: are the conference videos published?
+
+
+Sessions file
+-------------
+
+The sessions file contains the conference sessions, i.e. the talks.
+
+Each talk has the following attributes:
+
+* ``title``: title of the talk
+* ``abstract``: talk abstract
+* ``slug``: talk slug - referenced in the schedule.Typically, the slug is
+  a slugified version of the title, followed by the slugified speaker name.
+* ``series``: the conference series, used for videos, e.g. ``Write the Docs PORTLAND``
+* ``series_slug``: the slug of the series, used for videos
+* ``year``: the year the talk was given, used for videos
+* ``youtubeId``: the Youtube ID of the talk video, if published already
+* ``speakers``: the speaker(s) for the talk, in a list of speakers with keys:
+    * ``name``: name of the speaker
+    * ``slug``: slug of the speaker
+    * ``twitter``: Twitter username
+    * ``website``: URL of the speaker's website
+
+
+Schedule file
+-------------
+
+The schedule file contains the conference schedule. This is a mix of conference
+talks, and other agenda items like "Switch Speakers" or "Lunch Break".
+
+If you're writing a new conference schedule, it may be easier to start from
+the schedule file of the same conference as last year, as they tend to be
+quite similar.
+
+For conferences with a writing day, there must be a ``writing_day`` key. Then,
+a schedule for each main conference day, in the form of ``day1``, ``day2``,
```

Comment:
in one of the previous files, "day1" and so on is used to indicate whatever days, so for example day 1 would be the hike/boat, but in the schedule file day1 means day 1 of talks and things. This might cause some confusion, not sure how we can fix this... maybe standardize on the day numbering so that we all know what day 1 means everywhere?

Reply:
This is difficult, in part due to legacy, but I will change the schedule, which is still very new, to say "talks-day1" etc. That'll reduce some of the confusion.